src/TimeService.Business/Helpers/Emails/EmailSender.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using LT.DigitalOffice.Kernel.Helpers.TextHandlers.Interfaces;
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.Models.Broker.Responses.TextTemplate;
+using LT.DigitalOffice.TimeService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.TimeService.Data.Interfaces;
+
+namespace LT.DigitalOffice.TimeService.Business.Helpers.Emails
+{
+  public class EmailSender
+  {
+    private readonly IWorkTimeRepository _workTimeRepository;
+    private readonly IUserService _userService;
+    private readonly IEmailService _emailService;
+    private readonly ITextTemplateService _textTemplateService;
+    private readonly ITextTemplateParser _parser;
+
+    private async Task NotifyAsync(
+      UserData user,
+      string locale)
+    {
+      IGetTextTemplateResponse textTemplate = await _textTemplateService
+        .GetAsync(TemplateType.EmptyUserWorktimes, locale, null);

```

Comment:
remove null, you have default in the method

Reply:
done